src/openforms/scss/admin/_app_overrides.scss

```diff
@@ -98,6 +98,31 @@ $djai-padding--tablet: 30px;
   padding-top: 1em;
 }
 
+.admin-login-divider {
+  text-align: center;
+  clear: both;
+  padding-top: 1em;
+
+  width: 100%;
+  text-align: center;
+  border-bottom: 1px solid #ccc;
+  line-height: 0.1em;
+  margin: 10px 0 10px;
+
+  span {
+    background: #fff;
+    padding: 0 10px;
+  }
+}
+
+/**
+ * mozilla-django-oidc-db
+ * hide the regular admin login form by default if OIDC is enabled
+ */
+#login-form:has(~ .admin-login-oidc) {
+  display: none;
+}
```

Comment:
I think I'd make these selectors more component-oriented with the proposed class names to follow BEM methodology.

```scss
.admin-login-option {
  @include bem.modifier('username') {
    ~ .admin-login-option--oidc {
      display: none;
    }
  }
}
```

Reply:
But the rule at line 122 is meant to set `display: none;` on the `#login-form` itself, whereas your suggestion hides the OIDC option instead (if I understand it correctly)?

Reply:
I didn't try out the actual CSS, so yeah, probably should throw in a `:has(...)` in there! My point was more about using the BEM methodology :)